catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -145,19 +151,36 @@ public UpdateResponse version(UpdateResponse updateResponse)
         }
         setSkipFlag(updateResponse);
 
+        if (LOGGER.isTraceEnabled()) {
+            LOGGER.trace("Versioning updated metacards: " + updateResponse.getUpdatedMetacards());
```

Comment:
We could just put the number (or we could add in the number additionally). My reasoning behind logging the full metacard (hopefully they know how to represent themselves toString), is that on test/production systems if we're ever trying to debug a historian problem It might be nice to be able to get all the metacards information.

As an alternate solution, I could add a log at `debug` that prints the number, and then leave this full info at trace?